src/plugins/Scrollbar.js

```diff
@@ -28,7 +28,7 @@ var UNSCROLLABLE_STATE = {
   scrollable: false,
 };
 
-var FACE_MARGIN = parseInt(cssVar('scrollbar-face-margin'), 10);
+var FACE_MARGIN = parseInt(cssVar('--scrollbar-face-margin'), 10);
```

Comment:
The latest version of `postcss-custom-properties` requires you to specify variables with `--` prefixed